site/content/integrate/apps/_index.md

```diff
@@ -41,20 +41,20 @@ Read the [JavaScript start guide]({{< ref "quick-start-js" >}}) or [Go quick sta
 
 ### When would you build an app vs a custom slash command vs a webhook vs a plugin?
 
-That depends on your use case, as they each have benefits. Apps can be advertised, shared, and installed by others via the Marketplace. Comparatively, a custom slash command requires some configuration and is less interactive.
+That depends on your use case, as they each have benefits. Apps provide a wider scope for UX functionality, don't include any configuration on the users' side, and are language-agnostic. Comparatively, a custom slash command requires some configuration and is less interactive.
 
-The built-in incoming webhook is rigid and requires specific fields to be posted to it's URL, but some external systems don't allow the customization of the fields included in the webhook payload. Compared to built-in webhooks, An App can be customized to receive an HTTP webhook posting from another system, and can process the incoming data then augment it or make an actionable message in a channel.
+The built-in incoming webhook is rigid and requires specific fields to be posted to its URL, but some external systems don't allow the customization of the fields included in the webhook payload. Compared to built-in webhooks, An app can be customized to receive an HTTP webhook posting from another system, and can process the incoming data then augment it or make an actionable message in a channel.
 
-A plugin should be used when you need to alter the React UX in Mattermost or have an application that requires low-latency with the server (such as replacing characters in any message). Currently, plugins have several UX hooks that Apps cannot access, however we plan to add/migrate more UX hooks into the Apps Framework over time.
+A plugin should be used when you need to alter the UX in Mattermost or have an application that requires low-latency with the server (such as replacing characters in any message). Currently, plugins have several UX hooks that apps cannot access, however we plan to add/migrate more UX hooks into the Apps Framework over time.
 
 ### What's the difference between the apps framework and the plugin framework?
 
-The App framework provides a few differences from plugins.
+The app framework provides a few differences from plugins.
```

Comment:
Didn't we agree one
```suggestion
The Apps Framework provides a few differences from plugins.
```
?

Reply:
@hanzei I wish there was a linter for Docs. ;)

Reply:
I agree. 1/5 "Apps Framework" seems right as a proper noun